README.md

```diff
@@ -113,6 +113,24 @@ windows-*`].
 
 📚 If you're interested in more discussion of alternatives, check out [#4].
 
+## Development
+
+![YAML](https://img.shields.io/static/v1?style=for-the-badge&message=YAML&color=CB171E&logo=YAML&logoColor=FFFFFF&label=)
+![Bash](https://img.shields.io/static/v1?style=for-the-badge&message=Bash&color=4EAA25&logo=GNU+Bash&logoColor=FFFFFF&label=)
+
+This is a GitHub Action, so we inevitably use YAML. Make sure you quote the
+right things! To get started editing, you can use [GitHub.dev] for simple
+changes. For more complicated stuff, it's recommended to use [GitHub Codespaces]
+or a full environment. You can't run shellcheck on GitHub.dev yet!
+
+You can't really test the action locally. Or at least, we haven't defined a test
```

Comment:
I think one could use nektos/act. But I think this doc should focus on ways that you *can* hack away, not ways that you can't (or we don't know how to).